src/logic/addressBook/store/selectors/index.ts

```diff
@@ -64,19 +64,6 @@ export const addressBookEntryName = createSelector(
   },
 )
 
-export const addressBookName = createSelector(
```

Comment:
this was a duplicate of `addressBookEntryName` selector

Reply:
It's not a complete duplicate. It returns a name or undefined. `addressBookEntryName` returns a name _or a fallback value_.
Not ideal with the code duplication but we don't want the fallback value in some UIs.